Sources/mpc-core-kit-swift/mpcCoreKitSwift.swift

```diff
@@ -382,10 +381,10 @@ public struct MpcCoreKit  {
         guard let oauthKey = self.oauthKey else {
             throw "invalid oauth key"
         }
-        guard let uid = "\(oauthKey)_\(self.option.Web3AuthClientId)".data(using: .utf8)?.sha256() else {
+        guard let uid = try "\(oauthKey)_\(self.option.Web3AuthClientId)".data(using: .utf8)?.sha3(varient: Variants.KECCAK256 ) else {
```

Comment:
Might be better to convert it to hex here first if this will not be needed in its' current form and instead remove the conversion later?
```suggestion
        guard let uid = try "\(oauthKey)_\(self.option.Web3AuthClientId)".data(using: .utf8)?.sha3(varient: Variants.KECCAK256 )?.toHexString() else {
```

Reply:
done